src/pages/AnimeSub/main.ts

```diff
@@ -0,0 +1,101 @@
+import { pageInterface } from '../pageInterface';
+
+export const AnimeSub: pageInterface = {
+  name: 'AnimeSub',
+  domain: 'https://animesub.lt',
+  languages: ['Lithuanian'],
+  type: 'anime',
+  isSyncPage() {
+    if (window.location.href.split('/')[6] !== '') return true;
+    return false;
+  },
+  sync: {
+    getTitle(url) {
+      return AnimeSub.overview!.getTitle!(url);
+    },
+    getIdentifier() {
+      return getId()!;
+    },
+    getOverviewUrl() {
+      return `${window.location.href.split('#')[0]}`;
+    },
+    getEpisode() {
+      return Number(j.$('.scroller ul li.active .info').text().trim().split(' ')[0] || 1);
+    },
+    uiSelector(selector) {
+      j.$('.viewer').after(j.html(selector));
+    },
+    nextEpUrl(url) {
+      const nextEp = j.$('.scroller ul li.active').next();
+      if (!nextEp.length) return '';
+      return `${AnimeSub.sync.getOverviewUrl(url)}${nextEp.find('a').attr('href')}`;
+    },
+    getMalUrl(provider) {
+      return AnimeSub.overview!.getMalUrl!(provider);
+    },
+  },
+  overview: {
+    getTitle() {
+      return j.$('body > div.container.post-anime > div > div:nth-child(1) > h1').text().trim();
+    },
+    getIdentifier() {
+      return getId()!;
+    },
+    uiSelector(selector) {
+      j.$('.viewer').after(j.html(selector));
+    },
+    getMalUrl(provider) {
+      if (provider === 'MAL') {
```

Comment:
If a mal url exist then you can always return it, as every provider understands malurls